start-job/scripts/otto.py

```diff
@@ -7,7 +7,7 @@ def female_clothes() -> List[dict]:
     filter = "?nachhaltigkeit=alle-nachhaltigen-artikel"
 
     path_2_category = {
-        "blazer": "SWEATER",
+        "blazer": "SUIT",
```

Comment:
I also used `SUIT` as a mapping for blazer in asos mappings: 
https://github.com/calgo-lab/green-db/blob/4aff1c070ad0282e692423a408b61dfc0d72fa0c/start-job/data/asos_product_types.json#L688-L695